omod/src/main/java/org/openmrs/module/owa/impl/DefaultAppManager.java

```diff
@@ -108,10 +108,12 @@ public void installApp(File file, String fileName, String rootPath) throws IOExc
 
 				String dest = getAppFolderPath() + File.separator + deployedName;
 
-				Unzip unzip = new Unzip();
-				unzip.setSrc(file);
-				unzip.setDest(new File(dest));
-				unzip.execute();
+				try {
+					net.lingala.zip4j.core.ZipFile zipFile = new net.lingala.zip4j.core.ZipFile(file);
```

Comment:
There is already a ZipFile used in this file so here I'm specifying that I want the ZipFile from that package.